neo4j/internal/pool/server_test.go

```diff
@@ -106,23 +106,23 @@ func TestServer(ot *testing.T) {
 		conns := make([]*testutil.ConnFake, 3)
 		now := time.Now()
 		for i := range conns {
-			c := &testutil.ConnFake{Birth: now}
+			c := &testutil.ConnFake{Birth: now, ForceResetHook: func() {}}
 			conns[i] = c
 			s.registerBusy(c)
 			s.returnBusy(c)
 		}
 
-		// Let the conn in the middle be too old
+		// Let the connection in the middle be too old
 		conns[1].Birth = now.Add(-20 * time.Second)
 		s.removeIdleOlderThan(context.Background(), now, 10*time.Second)
 		assertSize(t, s, 2)
 
 		// Should be able to borrow twice
-		b1 := s.getIdle()
+		b1 := s.getIdle(nil, 0)
```

Comment:
should this be `context.Background()` instead of `nil` for style-points or does it not really matter in tests?

(many more like this, which I will not comment on)